srcpkgs/ugdb/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'ugdb'
 pkgname=ugdb
-version=0.1.9
+version=0.1.10
 revision=1
 build_style=cargo
+depends="gdb"
```

Comment:
Not sure I'm in favor, I usually use `ugdb` with `arm-none-eabi-gdb`.

Reply:
Not sure about this one either. Then again, we possible can trust in the user to point `ugdb` to some usable `gdb` binary, right?

Reply:
Yep, exactly. I'd prefer to leave without any `depends`

Reply:
Alright